tests/units/test_init.rb

```diff
@@ -45,8 +45,7 @@ def test_git_init
   def test_git_init_bare
     in_temp_dir do |path|
       repo = Git.init(path, :bare => true)
-      assert(File.directory?(File.join(path, '.git')))
-      assert(File.exist?(File.join(path, '.git', 'config')))
+      assert(File.exist?(File.join(path, 'config')))
```

Comment:
`Git.init` should create the new, bare repository in the directory given by path and not in the .git subdirectory of path. This was a bug.